ui/scripts/movie.js

```diff
@@ -109,6 +123,31 @@ export default class Movie {
         return conteiner;
     }
 
+    // expects countries object that has name and flag fields, class  
+    getFlags(counries, name) {
+        let flags = "";
+
+        if (counries.length !== 0) {
+            const list = document.createElement("ul");
+            list.classList.add(name);
+
+            counries.map(country => {
+                if (country.flag) {
+                    flags += `<li class="country-flag">${country.flag}<span class="country-name-tooltip">${country.name}</span></li>`;
+                } else {
+                    // handle null flag
+                    // &#127987; adds white flag symbol
+                    flags += `<li class="country-flag">&#127987;<span class="country-name-tooltip">${country.name}</span></li>`;
```

Comment:
in general, you should not repeat yourself, so instead of maintaining the format string for the list element here and above, you could:

```
flag = country.flag || defaultFlag;
flags += `<li class="country-flag">{flag}<span class="country-name-tooltip">${country.name}</span></li>`;
```